modules/adtelligentBidAdapter.js

```diff
@@ -0,0 +1,246 @@
+const BIDDER_CODE = 'adtelligent';
+const OUTSTREAM = 'outstream';
+const DISPLAY = 'display';
+const syncsCache = {};
+
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: ['onefiftytwomedia'],
+  supportedMediaTypes: [VIDEO, BANNER],
+  isBidRequestValid: function (bid) {
+    return !!utils.deepAccess(bid, 'params.aid');
+  },
+  getUserSyncs: function (syncOptions, serverResponses) {
+    const syncs = [];
+
+    function addSyncs(bid) {
+      const uris = bid.cookieURLs;
+      const types = bid.cookieURLSTypes || [];
+
+      if (Array.isArray(uris)) {
+        uris.forEach((uri, i) => {
+          const type = types[i] || 'image';
+
+          if ((!syncOptions.pixelEnabled && type == 'image') ||
+            (!syncOptions.iframeEnabled && type == 'iframe') ||
+            syncsCache[uri]) {
+            return;
+          }
+
+          syncsCache[uri] = true;
+          syncs.push({
+            type: type,
+            url: uri
+          })
+        })
+      }
+    }
+
+    if (syncOptions.pixelEnabled || syncOptions.iframeEnabled) {
+      utils.isArray(serverResponses) && serverResponses.forEach((response) => {
+        if (response.body) {
+          if (utils.isArray(response.body)) {
+            response.body.forEach(b => {
+              addSyncs(b);
+            })
+          } else {
+            addSyncs(response.body)
+          }
+        }
+      })
+    }
+    return syncs;
+  },
+  /**
+   * Make a server request from the list of BidRequests
+   * @param bidRequests
+   * @param bidderRequest
+   */
+  buildRequests: function (bidRequests, bidderRequest) {
+    return {
+      data: bidToTag(bidRequests, bidderRequest),
+      bidderRequest,
+      method: 'GET',
+      url: URL
+    };
+  },
+
+  /**
+   * Unpack the response from the server into a list of bids
+   * @param serverResponse
+   * @param bidderRequest
+   * @return {Bid[]} An array of bids which were nested inside the server
+   */
+  interpretResponse: function (serverResponse, {bidderRequest}) {
+    serverResponse = serverResponse.body;
+    let bids = [];
+
+    if (!utils.isArray(serverResponse)) {
+      return parseRTBResponse(serverResponse, bidderRequest);
+    }
+
+    serverResponse.forEach(serverBidResponse => {
+      bids = utils.flatten(bids, parseRTBResponse(serverBidResponse, bidderRequest));
+    });
+
+    return bids;
+  }
+};
+
+function parseRTBResponse(serverResponse, bidderRequest) {
+  const isInvalidValidResp = !serverResponse || !utils.isArray(serverResponse.bids);
+
+  const bids = [];
+
+  if (isInvalidValidResp) {
+    const extMessage = serverResponse && serverResponse.ext && serverResponse.ext.message ? `: ${serverResponse.ext.message}` : '';
+    const errorMessage = `in response for ${bidderRequest.bidderCode} adapter ${extMessage}`;
+
+    utils.logError(errorMessage);
+
+    return bids;
+  }
+
+  serverResponse.bids.forEach(serverBid => {
+    const requestId = findIndex(bidderRequest.bids, (bidRequest) => {
```

Comment:
Any strong reason to use find-index from core-js. We have a white list of core-js modules here https://github.com/prebid/Prebid.js/blob/master/allowedModules.js#L2 
Can you update the logic ? 
Also, we have a custom eslint rule but not sure why it did not catch this import. I will look into this.

Reply:
nope, just left from previous dev =)

Reply:
so, can i use Array.prototype.findIndex ?

Reply:
@GeneGenie Array.prototype.findIndex does not support Internet explorer and hence the previous dev used core-js to support IE. I would suggest to use `find` from core-js

Here is the updated code which worked fine on my test page. Do test in on your end as well.

```
const request = find(bidderRequest.bids, (bidRequest) => {
      return bidRequest.bidId === serverBid.requestId;
    });

    if (serverBid.cpm !== 0 && request !== undefined) {
      const bid = createBid(serverBid, getMediaType(request));

      bids.push(bid);
    }
```

Reply:
fixed